src/components/DatePicker/docs/DatePickerWiringExample.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { DatePicker } from '../';
+import { classes } from './DatePickerWiringExample.st.css';
+
+export const DatePickerWiringExample = () => {
+  return (
+    <DatePicker
+      className={classes.root}
+      value={new Date()}
+      onChange={() => {}}
```

Comment:
Let's add a state to this example, so that the `onChange` will actually change the selected date